dashboard-app/src/dashboard/ByTime/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import moment from 'moment';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import { H1 } from '../../components/Headings';
+import { CommunityBusinesses } from '../../api';
+import DataTable from '../../components/DataTable';
+import { DataTableProps } from '../../components/DataTable/types';
+import useRequest from '../../util/hooks/useRequest';
+import { DurationUnitEnum } from '../../types';
+import DateRange from '../../util/dateRange';
+import { timeLogsToTable } from './helper';
+import UtilityBar from '../../components/UtilityBar';
+
+export default () => {
+  const [unit, setUnit] = useState(DurationUnitEnum.HOURS);
+  const [fromDate, setFromDate] = useState(moment().subtract(1, 'year').add(1, 'month').toDate());
+  const [toDate, setToDate] = useState(moment().toDate());
+  const [tableProps, setTableProps] = useState<DataTableProps>();
+
+  const { error, loading, data } = useRequest({ // tslint:disable-line
```

Comment:
I'm not expecting this to be dealt with in this PR, but since all tables rely ultimately on the same API call, it probably makes sense to make the call only once, keep one copy of the data in memory, and then pass it down to the pages/tables to process as they wish.

Reply:
All data from all of time? This makes a call when the `from` & `to` dates are changes. I think this is a good way of handling the data.

Reply:
There are ways to deal with this, for example, fetching some default amount of data initially, and then updating it with an additional API call if out-of-range data is requested by the user. But this is fairly involved, which is why I'm not suggesting any of this for this PR.

Reply:
@eliascodes do you want to break this out into an issue? Or are you happy for me to resolve this comment?